### PR TITLE
Validate format of dates matches DD.MM.YYYY

### PR DESCRIPTION
**Pull request by bellisk**
Also, translate dataset description using the scheming helper, so that it's more robust in case the description is returned as a dict instead of a string.

### CODE REVIEW

File: ckanext/switzerland/tests/test_controller.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def setup(self):
         # create a valid DCAT-AP Switzerland compliant dataset
         self.dataset = {
             'coverage': '',
-            'issued': '2015-09-08T00:00:00',
+            'issued': '08.09.2015',
```
Comment (sabinem) [inline ckanext/switzerland/tests/test_controller.py:41]: The dates must be in ISO-Format in the API and XML. So I am not quite sure whether that will still be the case with that change. Did you check this?

File: ckanext/switzerland/presets.json [hunk 2]
```diff
@@ -146,7 +146,7 @@
       "values": {
         "display_snippet": "date.html",
         "form_snippet": "date.html",
-        "validators": "scheming_required date_string_to_timestamp",
+        "validators": "scheming_required ogdch_date_string_format date_string_to_timestamp",
```
Comment (sabinem) [inline ckanext/switzerland/presets.json:149]: An additional vaildator may affect the harvesters. Did you test run them?

**Review by sabinem — COMMENTED** [review]
I am just not sure about the effects of your change on:
- harvesting
- the rdf export
- the api export
The dates in the API and XML downloads should be ISO Dates if they belong to the DCAT-AP-CH fields. Please make sure that all these things are taken care of. If that is the case I will merge the change.

File: ckanext/switzerland/tests/test_controller.py (hunk 1 above)
Comment (bellisk) [reply ckanext/switzerland/tests/test_controller.py:41]: The format in the API is handled by the output_validators, not the input validators, and the XML export is handled by ckanext-dcatapchharvest, so neither of them is affected by this change.

**Pull request merged**